src/interraction.asm

```diff
@@ -421,11 +433,49 @@ section .text
         cmp eax, [endFruitTime]
         jne .stillFruitTime
 
-        int3
         mov si, MazeModelBuffer
         add si, FRUIT_TILE_POS
         mov byte[si], 0x0F
         call replaceFruit
 
         .stillFruitTime:
+        ret
+
+    lifeManagement: 
+        cmp word[scoreForLife], 10000
+        jl .noNewLife
```

Comment:
Should be `jb` instead of `jl` since the value is always positive.

Reply:
Corrected